angular-rails-app/Gemfile

```diff
@@ -32,6 +32,8 @@ gem 'ngannotate-rails'
 gem 'sdoc', '~> 0.4.0', group: :doc
 # pagination.
 gem 'will_paginate', '~> 3.1.0'
+# ui directives for Angular
+gem 'angular-ui-bootstrap-rails'
```

Comment:
Cool. For those following along, you'll need to run the following to make sure your container is up to date;

```bash
docker-compose build web && docker-compose restart 
```